src/hwProvider.ts

```diff
@@ -43,6 +44,12 @@ export class HWProvider {
     }
 
     async getTransport(): Promise<Transport> {
+        let webBLESupported = await TransportWebBLE.isSupported();
```

Comment:
Do we want this as the primary transport?

Reply:
Need to think on how to switch between transporters dynamically if something goes wrong